pulp-glue/pulp_glue/common/context.py

```diff
@@ -151,6 +151,26 @@ def _patch_api_spec(self) -> None:
                             and parameter["schema"]["type"] == "string"
                         ):
                             parameter["schema"] = {"type": "array", "items": {"type": "string"}}
+        if self.has_plugin(PluginRequirement("core", max="99.99.0")):
+            # https://github.com/pulp/pulpcore/issues/3634
+            for operation_id, (method, path) in self.api.operations.items():
```

Comment:
You think we should look into moving this method into it's own file? `context.py` will grow quite large as we discover more schema bugs. Also, can plugins that are not apart of this package contribute to this method?

Reply:
No, they cannot, and that is a valid concern.